pages/fullscreen/playback/index.vue

```diff
@@ -50,8 +50,7 @@ export default Vue.extend({
         keyboard: true,
         a11y: true
       } as SwiperOptions,
-      swiper: undefined as Swiper | undefined,
-      previousAppBarOpacity: 0
```

Comment:
What if I'm accessing this page from an item details page, for instance? (As it has transparency)

Reply:
It seems every page that needs it set their transparency at activation, so it the player (or any page) shouldn't have to keep what was previously set imo :)